.github/workflows/sub_gradle_test_and_build.yml

```diff
@@ -113,6 +114,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Set up Gradle properties
+        run: |
+          echo "kotlin.daemon.jvmargs=-Xmx2g" >> gradle.properties
```

Comment:
Does this fix any issues? Please comment.

Reply:
It was added to resolve OutOfMemory issue of CodeQL Analysis execution.
Error message:
`[2023-08-23 14:11:36] [autobuild] Execution failed for task ':integration-tests:compileTestKotlin'.
  [2023-08-23 14:11:36] [autobuild] > A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
  [2023-08-23 14:11:36] [autobuild]    > Not enough memory to run compilation. Try to increase it via 'gradle.properties':
  [2023-08-23 14:11:36] [autobuild]      kotlin.daemon.jvmargs=-Xmx<size>`

https://github.com/stellar/java-stellar-anchor-sdk/actions/runs/5950927025/job/16143863193